content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -210,6 +210,7 @@ currently include:
  * `node.kubernetes.io/memory-pressure`: Node has memory pressure.
  * `node.kubernetes.io/disk-pressure`: Node has disk pressure.
  * `node.kubernetes.io/network-unavailable`: Node's network is unavailable.
+ * `node.kubernetes.io/unschedulable`: Node is unschedulable. (1.10 or later)
  * `node.cloudprovider.kubernetes.io/uninitialized`: When kubelet is started
    with "external" cloud provider, it sets this taint on a node to mark it
    as unusable. When a controller from the cloud-controller-manager initializes
```

Comment:
Is `TaintBasedEvictions` still alpha?

Reply:
Yes, `TaintBasedEvictions` is still alpha right now .